src/app/public-view/components/another-blog-articles/another-blog-articles.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="row">
+  <div class="col-md-4 p-2" *ngFor="let blog of blogsTitleImages">
+    <div class="card">
+      <a href="/blog/{{blog.id}}" class="thumb">
```

Comment:
Nedá sa tu miesto href použiť routerLink="/blog/{{blog.id}}"?
Takto sa musí refreshnúť stránka

Reply:
vies tam nechat <a> element, len nedas href, ale pouzijes tuto triedu na element https://getbootstrap.com/docs/4.4/utilities/stretched-link/ a staci ti dat (click) na <a> a presmerovanie spravis cez funkciu v komponente, kde pouzijes router... vyhoda v tomto je, ze ti neurobi refresh stranky

Reply:
To bolo prve riesenie ake som spravil ale neprebehlo to podla predstav, zmenila sa url adresa ale data sa nenacitali spravne